components/tile.scss

```diff
@@ -0,0 +1,4 @@
+$fd-icons-path: "../scss/icons/";
+
+@import "../scss/components/tile";
+@import "../scss/icons/icon";
```

Comment:
Does the Tile component need icons?

Reply:
please see the screenshot:
![Screen Shot 2019-08-02 at 4 08 57 PM](https://user-images.githubusercontent.com/41170611/62396002-e4573b80-b53f-11e9-957f-b7c088fa16a3.png)
the overflow icon inside button